services/shhext/chat/crypto/ethereum_crypto.go

```diff
@@ -0,0 +1,196 @@
+package crypto
```

Comment:
Not a fan of skipping error handling in many methods here :P

Reply:
I will add error handling once we go through the double ratchet library a bit more methodically (this is a copy and paste of the default implementation with just the diffie helman exchange swapped out)